pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools", "wheel"]
```

Comment:
Setuptools' docs used to say this was necessary, but they clarified it was a bad recommendation.